bank_statement_import_invoices_fix/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of importing invoices as bank statement lines.
+It invert values of "communication" and "reference" fields
+
```

Comment:
This sentence could help to explain why:

When lines are imported the reference goes to the communication field (instead of reference of the bank statement line) in order to have a default proposal from the system during the reconciliation process.